src/main/java/ch/njol/skript/lang/VariableString.java

```diff
@@ -587,18 +582,22 @@ public String toString(@Nullable Event event, boolean debug) {
 	 * @return List<String> of all possible super class code names.
 	 */
 	public List<String> getDefaultVariableNames(String variableName, Event event) {
```

Comment:
This method returns either a mutable or immutable list, we should probably just stick with one.

Reply:
Where is the immutable list?

Reply:
Collectors#toList is not guaranteed to return a mutable list.

Reply:
It's not a worry here though. It's returning a list of generated default variable hints that are to match the database. The list is never modified.

Reply:
Could be immutable then, but I'll approve since that would only be a minor improvement